numpy/core/src/umath/ufunc_object.c

```diff
@@ -2719,6 +2719,20 @@ reducelike_promote_and_resolve(PyUFuncObject *ufunc,
      * cannot quite handle the correct ops (e.g. a NULL first item if `out`
      * is NULL) so we pass `arr` instead in that case.
      */
+    /*
```

Comment:
I'd keep the comment on `ops` above together with the definition, i.e., below this stanza.  Alternatively, since this is only executed when `out` is defined, and since only links to `out` are used, maybe put this part inside the `if (out != NULL)` below?

Reply:
Thanks, moved the `ops` initialization in between these comments again.  Moving around more may help, but didn't really see it being much better.